### PR TITLE
make the check circle and its color themeable

### PR DESCRIPTION
**Pull request by M3psipax**
Please accept this PR to expose two new style attributes in order to be able to change the color of the circle which is used to select an image.

"item.checkCircle.backgroundColor"
"item.checkCircle.borderColor"

### CODE REVIEW

**Comment by M3psipax** [comment]
the build has an internal server error. This doesn't have anything to do with my changes, correct?

File: matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java [hunk 1]
```diff
@@ -84,7 +85,10 @@ private void init(Context context) {
         mStrokePaint.setStyle(Paint.Style.STROKE);
         mStrokePaint.setXfermode(new PorterDuffXfermode(PorterDuff.Mode.SRC_OVER));
         mStrokePaint.setStrokeWidth(STROKE_WIDTH * mDensity);
-        mStrokePaint.setColor(Color.WHITE);
+        TypedArray ta = getContext().getTheme().obtainStyledAttributes(new int[]{R.attr.item_checkCircle_borderColor});
+        int color = ta.getColor(0, 0);
```
Comment (gejiaheng) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java]: Better pass a default value which is `@color/zhihu_item_checkCircle_borderColor` cause someone may derive a custom theme from a system built-in theme.

File: matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java [hunk 2]
```diff
@@ -188,7 +192,11 @@ private void initBackgroundPaint() {
             mBackgroundPaint = new Paint();
             mBackgroundPaint.setAntiAlias(true);
             mBackgroundPaint.setStyle(Paint.Style.FILL);
-            mBackgroundPaint.setColor(Color.parseColor("#1E8AE8"));
+            TypedArray ta = getContext().getTheme()
+                    .obtainStyledAttributes(new int[]{R.attr.item_checkCircle_backgroundColor});
+            int color = ta.getColor(0, 0);
```
Comment (gejiaheng) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java]: Same here, pass a default value which is `@color/zhihu_item_checkCircle_backgroundColor`

Comment (M3psipax) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java]: You mean, it should look something like this:

` int color = ta.getColor(0, ResourcesCompat.getColor(getResources(), R.color.zhihu_item_checkCircle_borderColor, getContext().getTheme()));`

Yes?

Comment (gejiaheng) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java]: Yes, exactly, unless you think this has any problems.

Comment (M3psipax) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java]: Nope, it's okay. I'll have it ready as soon as possible. Do I have to create a new pull request after I've pushed the changes to my fork?

Comment (gejiaheng) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java]: I think this PR will update automatically if you push new code to the same branch which is `M3psipax:master`.

Comment (M3psipax) [inline matisse/src/main/java/com/zhihu/matisse/internal/ui/widget/CheckView.java]: Okay, it's done

**Pull request merged**